### PR TITLE
[expo-image-picker] Fix getCameraPermissionsAsync on web

### LINKED ISSUE #7470: [ Web ] expo-image-picker  getCameraPermissionsAsync not a function
### Environment

```shell
  Expo CLI 3.15.5 environment info:
    System:
      OS: macOS 10.15.3
      Shell: 5.7.1 - /bin/zsh
    Binaries:
      Node: 13.11.0 - ~/.nvm/versions/node/v13.11.0/bin/node
      npm: 6.13.7 - ~/.nvm/versions/node/v13.11.0/bin/npm
      Watchman: 4.9.0 - /usr/local/bin/watchman
    IDEs:
      Android Studio: 3.4 AI-183.5429.30.34.5452501
      Xcode: 11.4/11E146 - /usr/bin/xcodebuild
    npmPackages:
      expo: ~36.0.2 => 36.0.2
      react: ^16.13.1 => 16.13.1
      react-native: https://github.com/expo/react-native/archive/sdk-36.0.0.tar.gz => 0.61.4
    npmGlobalPackages:
      expo-cli: 3.15.5
```

Target: web.

### Steps to Reproduce

Use `ImagePicker.getCameraPermissionsAsync()` as per documentation in a browser environment.

### Expected Behavior

To ask for permissions.

### Actual Behavior

Function does not exist. I ended up using the `Permissions` module directly which works fine.

### Reproducible Demo

[Snack](https://snack.expo.io/@mentor/intrigued-cake).

### PR DESCRIPTION
**Pull request by IjzerenHein**
# Why

This PR fixes the expo-image-picker `getCameraPermissionsAsync` function on web.

Fixes #7470

# How

There was a typo in the web-implementation, causing this function to throw an error.

# Test Plan

**Before fix:**

image

**After fix:**

image

### CODE REVIEW

**Review by bbarthec — APPROVED** [review]

**Comment by bbarthec** [comment]
CHANGELOG in `expo-image-picker` should be updated 😉

**Comment by IjzerenHein** [comment]
> CHANGELOG in `expo-image-picker` should be updated 😉

Of course, will do :D

File: packages/expo-image-picker/CHANGELOG.md [hunk 1]
```diff
@@ -7,3 +7,5 @@
 ### 🎉 New features
 
 ### 🐛 Bug fixes
+
+- Fixed exception when calling `getCameraPermissionsAsync` on Web. ([#7498](https://github.com/expo/expo/pull/7498) by [@IjzerenHein](https://github.com/IjzerenHein))
```
Comment (bbarthec) [inline packages/expo-image-picker/CHANGELOG.md]:
```suggestion
- Fixed exception when calling `ImagePicker.getCameraPermissionsAsync` on Web. ([#7498](https://github.com/expo/expo/pull/7498) by [@IjzerenHein](https://github.com/IjzerenHein))
```

Comment (bbarthec) [reply packages/expo-image-picker/CHANGELOG.md]: Summon @tsapeta 🤔 Do you think we should always use `Module.function` notation in these `CHANGELOG`s?

**Pull request merged**